pkg/dropbear

```diff
@@ -13,25 +13,30 @@ serviceflags="--down --log"
 zlib
 
 [build]
-[ -n "$CROSS_COMPILE" ] && xconfflags="--host=$($CC -dumpmachine)"
-CPPFLAGS="-D_GNU_SOURCE -D_BSD_SOURCE" \
-CFLAGS="$optcflags" \
-LDFLAGS="$optldflags" \
-  ./configure -C --prefix="$butch_prefix" $xconfflags
-
-# TODO : edit generated options.h
+[ -n "$CROSS_COMPILE" ] && \
+  xconfflags="--host=$($CC -dumpmachine) \
+  --with-sysroot=$butch_root_dir"
```

Comment:
was there a need to add --with-sysroot ?

Reply:
I just stole it from the /src/KEEP/pkg_skel/autoconf. Figured it was part of the skel for a reason. I'll remove it.